ui/src/services/admin/admin.service.ts

```diff
@@ -130,7 +130,7 @@ export class AdminService extends BaseService {
         });
     }
 
-    public importFrom(file: File, progressFunction: (progressEvent: any) => void): Promise<void> {
+    public importFrom(file: string | File, progressFunction: (progressEvent: any) => void): Promise<void> {
```

Comment:
When would this be a string?

Reply:
the changes to make file a "string | File" type are all due to `FileUpload` component. The "filename" parameter in the `FileUpload` component has that type, therefore, for compilation to pass we have to set that type everywhere

Reply:
the compilation started to fail after I modified that setMultiState function you comented before

Reply:
I see.  I wonder if we should instead cast the object to `File` when calling `setMultiState` instead?

Reply:
I think this is fine as is, no file is empty string and we keep the same type as required by FileUpload, win win